### PR TITLE
Change hardcoded image shape to newly defined `INPUT_SHAPE` variable

### LINKED ISSUE #27: remove hardcoded image size in boundary augmentation

### PR DESCRIPTION
**Pull request by laureho**
Fix #27 

Boundary augmentation in the `augment` function now takes the input layer shape of `model` instead of a hardcoded image shape

### CODE REVIEW

File: notebooks/C_CNN_Training_and_Validation.ipynb [hunk 1]
```diff
@@ -1,602 +1,602 @@
...
+   "execution_count": null,
+   "metadata": {
+    "id": "u-QTgXl7m1qF"
+   },
+   "outputs": [],
+   "source": [
+    "BATCH_SIZE = 64\n",
+    "BUFFER_SIZE = 20_000\n",
+    "TRAINING_EPOCHS = 100\n",
+    "BOUNDARY_AUGMENTATION = True"
```
Comment (quantumjot) [inline notebooks/C_CNN_Training_and_Validation.ipynb]: Probably worth adding:
`INPUT_SHAPE = (64, 64, 1)`

File: notebooks/C_CNN_Training_and_Validation.ipynb [hunk 2]
```diff
@@ -1,602 +1,602 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "KGgzMUfmm1qG"
+   },
+   "outputs": [],
+   "source": [
+    "img = K.layers.Input(shape=(64, 64, 1))\n",
```
Comment (quantumjot) [inline notebooks/C_CNN_Training_and_Validation.ipynb]: change this to:
`img = K.layers.Input(shape=INPUT_SHAPE)`

File: notebooks/C_CNN_Training_and_Validation.ipynb [hunk 3]
```diff
@@ -1,602 +1,602 @@
...
+   },
+   "outputs": [],
+   "source": [
+    "@augmentation_label_handler\n",
+    "def augment(img):\n",
+    "    if BOUNDARY_AUGMENTATION:\n",
+    "        # this will randomly simulate the cropping that occurs at the edge of\n",
+    "        # an image volume\n",
+    "\n",
+    "        vignette = np.ones((model.layers[0].input_shape[0][1], \n",
```
Comment (quantumjot) [inline notebooks/C_CNN_Training_and_Validation.ipynb]: then finally, we can use:
`vignette = np.ones(INPUT_SHAPE, dtype=np.float32)`

probably also changing the `30` in the random width to `INPUT_SHAPE // 2` while we're about it

**Review by quantumjot — CHANGES_REQUESTED** [review]
Looks good, but can you make these changes?

File: notebooks/C_CNN_Training_and_Validation.ipynb (hunk 1 above)
Comment (laureho) [reply notebooks/C_CNN_Training_and_Validation.ipynb]: ah ok, I thought about it but wasn't sure that's what you had in mind! will implement it now

**Review by quantumjot — APPROVED** [review]
LGTM!

**Pull request merged**